specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,702 @@
+        "cpu": {
+          "format": "double",
+          "description": "Required CPU in cores, e.g. 0.5",
+          "type": "number"
+        },
+        "memory": {
+          "description": "Required memory, e.g. \"250Mb\"",
+          "type": "string"
+        }
+      }
+    },
+    "SessionContainer": {
+      "description": "Container definitions for the Sessions of the Session Pool.",
+      "type": "object",
+      "properties": {
+        "image": {
+          "description": "Container image tag.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Custom container name.",
+          "type": "string"
+        },
+        "command": {
+          "description": "Container start command.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "args": {
+          "description": "Container start command arguments.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "env": {
+          "description": "Container environment variables.",
+          "type": "array",
+          "items": {
+            "$ref": "./CommonDefinitions.json#/definitions/EnvironmentVar"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "resources": {
+          "$ref": "#/definitions/SessionContainerResources",
+          "description": "Container resource requirements."
+        }
+      }
+    },
+    "SessionIngress": {
+      "description": "Session Pool Ingress configuration.",
+      "type": "object",
+      "properties": {
+        "targetPort": {
+          "format": "int32",
+          "description": "Target Port in containers for traffic from ingress",
+          "type": "integer"
+        }
+      }
+    },
+    "CustomContainerTemplate": {
+      "description": "Custom container configuration.",
+      "type": "object",
+      "properties": {
+        "registryCredentials": {
+          "description": "Private container registry credentials for containers used by the Sessions of the Session Pool.",
+          "$ref": "#/definitions/SessionRegistryCredentials"
+        },
+        "containers": {
+          "description": "List of container definitions for the Sessions of the Session Pool.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SessionContainer"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "ingress": {
+          "description": "SessionPool Ingress configuration.",
+          "$ref": "#/definitions/SessionIngress"
+        }
+      }
+    },
+    "SessionNetworkConfiguration": {
+      "description": "Session network configuration.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Network status for the Sessions.",
+          "type": "string",
+          "enum": [
+            "egressEnabled",
```

Comment:
Enum value should follow `PascalCasing` pattern like I have seen at multiple places like Status enum etc.

so, it should be `EgressEnabled`

Reply:
Thanks, you are right, I have changes the enums to be Pascal. For the containerType, I thought we were going with node-lts, but I found this PR https://msazure.visualstudio.com/One/_git/AAPT-Antares-Containerapps/pullrequest/9960874, and decided to make this CustomContainer and PythonLTS now